.gitlab-ci.yml

```diff
@@ -36,16 +36,19 @@ dotnet-package:
     - mv dotnet-agent-extract/bin/agent/datadog-trace-agent.exe $RELEASE_AGENT_DIR
     - echo "Creating nuget package"
     - echo "Packing nuspec file via arcane roundabout csproj process"
-    - dotnet pack dotnet/Datadog.AzureAppServices.csproj -p:NoBuild=true -p:NoDefaultExcludes=true -o package
-    - echo "Updating paths from Datadog.AzureAppServices to Datadog.Development.AzureAppServices for testing package"
-    - sed -i 's/Datadog.AzureAppServices/Datadog.Development.AzureAppServices/g' dotnet/content/applicationHost.xdt
-    - sed -i 's/v0_4_0/v100_5_15/g' dotnet/content/applicationHost.xdt
-    - sed -i 's/v0_4_0/v100_5_15/g' $RELEASE_AGENT_DIR/datadog.yaml
-    - sed -i 's/v0_4_0/v100_5_15/g' $RELEASE_AGENT_DIR/dogstatsd.yaml
+    - dotnet pack dotnet/Datadog.AzureAppServices.DotNet.csproj -p:NoBuild=true -p:NoDefaultExcludes=true -o package
+    - echo "Updating versions from v1_0_0 to v0_1_10 for testing package"
+    - sed -i 's/v1_0_0/v0_1_10/g' dotnet/content/applicationHost.xdt.dd
+    - sed -i 's/v1_0_0/v0_1_10/g' $RELEASE_AGENT_DIR/datadog.yaml
+    - sed -i 's/v1_0_0/v0_1_10/g' $RELEASE_AGENT_DIR/dogstatsd.yaml
+    - sed -i 's/v1_0_0/v0_1_10/g' $BASE_DIR/validate-w3wp-stopped.ps1
+    - echo "Updating paths from Datadog.AzureAppServices.DotNet to DevelopmentVerification.DdDotNet.Apm for testing package"
+    - sed -i 's/Datadog.AzureAppServices.DotNet/DevelopmentVerification.DdDotNet.Apm/g' $BASE_DIR/applicationHost.xdt.dd
+    - sed -i 's/Datadog.AzureAppServices.DotNet/DevelopmentVerification.DdDotNet.Apm/g' $BASE_DIR/validate-w3wp-stopped.ps1
     - mkdir $DEVELOPMENT_DIR
     - mv -v $RELEASE_DIR/* $DEVELOPMENT_DIR
     - rm -d -v $RELEASE_DIR
-    - dotnet pack dotnet/Datadog.Development.AzureAppServices.csproj -p:NoBuild=true -p:NoDefaultExcludes=true -o package
+    - dotnet pack dotnet/DevelopmentVerification.DdDotNet.Apm.csproj -p:NoBuild=true -p:NoDefaultExcludes=true -o package

```

Comment:
Is `DevelopmentVerification.DdDotNet.Apm` published as a nuget package? Why did we switch to such a cryptic name? Should it at least start with reserved `Datadog.*` prefix so it get's the official ✅ on the nuget page?

Reply:
Not a blocker since we can always change it later.

Reply:
No, I don't want users to even notice this package.
This is our testing and verification mechanism, at least until we find a way to use a private feed.